### PR TITLE
Enabled patch endpoint on registration

### PR DESCRIPTION
**Pull request by erikh360**

### CODE REVIEW

File: registrations/views.py [hunk 1]
```diff
@@ -104,17 +104,22 @@ def post(self, request, *args, **kwargs):
...
     serializer_class = RegistrationSerializer
+    lookup_field = 'id'
 
     def post(self, request, *args, **kwargs):
         # load the users sources - posting users should only have one source
         source = Source.objects.get(user=self.request.user)
         request.data["source"] = source.id
         return self.create(request, *args, **kwargs)
 
+    def patch(self, request, *args, **kwargs):
```
Comment (KaitCrawford) [inline registrations/views.py:120]: is this method necessary? doesn't the update mixin already provide this?

**Review by KaitCrawford — COMMENTED** [review]
looks good. 1 small query though

File: registrations/views.py (hunk 1 above)
Comment (erikh360) [reply registrations/views.py:120]: @KaitCrawford I get a `405 Method Not Allowed` error if I remove it.

Comment (KaitCrawford) [reply registrations/views.py:120]: odd. but ok

**Review by KaitCrawford — APPROVED** [review]
:+1:

File: registrations/views.py (hunk 1 above)
Comment (rudigiesler) [reply registrations/views.py:120]: The `UpdateModelMixin` gives us the `partial_update` and `update` methods. The `UpdateApiView` class is the one that provides the `put` and `patch` classes that uses these mixins: https://github.com/encode/django-rest-framework/blob/a251b9379200420062cad9e3c68fde7c0e6b3fdc/rest_framework/generics.py#L222-L231

Comment (KaitCrawford) [reply registrations/views.py:120]: Ah! thanks for the clarification Rudi

**Pull request merged**